pdns/pdns_recursor.cc

```diff
@@ -2566,6 +2567,9 @@ int serviceMain(int argc, char*argv[])
 
   showProductVersion();
   seedRandom(::arg()["entropy-source"]);
+  openssl_thread_setup();
+  openssl_seed();
```

Comment:
That nagging on the back of your mind is the stuff deterministic ECDSA fixes. Our core randomness need for openssl is generating keys, which indeed we do not do today in the recursor, but let's get this right from the start :)